packages/now-build-utils/src/detect-builders.ts

```diff
@@ -368,14 +381,11 @@ export async function detectBuilders(
 
   // Detect all builders for the `api` directory before anything else
   const builders = detectApiBuilders(files, options);
+  const { projectSettings = {} } = options;
+  const { outputDirectory, buildCommand, framework } = projectSettings;
 
-  if (detectorResult && detectorResult.buildCommand) {
-    const frontendBuilder = detectFrontBuilder(
-      detectorResult,
-      builders,
-      files,
-      options
-    );
+  if ((pkg && hasBuildScript(pkg)) || buildCommand || framework) {
```

Comment:
```suggestion
  if (hasBuildScript(pkg) || buildCommand || framework) {
```
This can be simplified here